__tests__/app.test.js

```diff
@@ -164,5 +159,160 @@ describe("post/api/articles/:articleid/comments", () => {
       });
   });
 });
+test("returns 404 error if article id not a number", () => {
+  const comment = {
+    username: "icellusedkars",
+    body: "Great article!",
+  };
+  return request(app)
+    .post("/api/articles/alex/comments")
+    .send(comment)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body).toEqual({ msg: "404, article not found" });
+    });
+});
+
+test("returns 404 error if no comment given", () => {
+  const comment = {
+    username: "icellusedkars",
+  };
+  return request(app)
+    .post("/api/articles/alex/comments")
+    .send(comment)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body).toEqual({ msg: "404, article not found" });
+    });
+});
```

Comment:
For this test the endpoint should be valid so the only thing you are testing is the comment not being given.